package.json

```diff
@@ -121,5 +121,13 @@
   },
   "dependencies": {
     "uuid": "^9.0.0"
+  },
+  "pnpm": {
+    "overrides": {
```

Comment:
>overrides [](http://example.com/codeflow?start=5&length=9)

Is it worth considering doing something to document the justification for these overrides so people in the future know why they are here and/or when they can remove them?